incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -262,12 +263,22 @@ func (r *HierarchyReconciler) syncLabel(log logr.Logger, nsInst *corev1.Namespac
 		return
 	}
 
+	// Pre-define depth label suffix
+	labelDepthSuffix := fmt.Sprintf(".tree.%s/depth", api.MetaGroup)
+
+	// Remove all existing HNC labels.
```

Comment:
We're not removing "all existing HNC labels," we're removing "all existing _depth_ labels."

Reply:
Ok. Done